src/ts/components/core/input/JsonInput.tsx

```diff
@@ -69,6 +70,7 @@ const JsonInput = ({
             onKeyDown={handleKeyDown}
             onBlur={handleBlur}
             autoComplete={autoComplete}
+            {...inputProps}
             {...others}
```

Comment:
Your goal here is to pass arbitrary props all the way through to the HTML, which makes sense. I'm just thinking about what happens if the user puts things in `inputProps` that are already supplied elsewhere. As written, if you have the same keys in both `inputProps` and `others`, `others` wins because it comes later, and I think that makes sense. But `inputProps` can also override the explicitly named props before it, like `value` or `onChange`, which would be rather awkward! We could of course say it's obvious you shouldn't do this, but if it's as easy as moving `{...inputProps}` up to the top of the prop list to prevent this pitfall maybe we should?

Reply:
Yes, good point.  I'll make the change